gammacat/collection.py

```diff
@@ -211,6 +225,23 @@ def process_lightcurves(self):
             log.info('Writing {}'.format(path))
             lightcurve.table.write(str(path), format='ascii.ecsv')
 
+    def process_datasets(self):
+        for filename in self.input_data.info_yaml_list:
+            folder = filename.parents[0]
+            info_data = load_yaml(filename)
+            status = info_data['data_entry']['status']
+            review = info_data['data_entry']['reviewed']
+            log.debug('Processing reference: {}'.format(info_data['reference_id']))
+            # TODO: This is if you want to make sure that all data are reviewed
```

Comment:
Currently, only datasets, which are marked as 'complete' in the corresponding info.yaml file, are added/copied to the collection.
In the end, we should add a check for 'reviewed == yes' as well. But now, everything is not reviewed, hence the collection would be empty.